plutus-tx-plugin/src/Language/PlutusTx/Compiler/Error.hs

```diff
@@ -98,3 +102,6 @@ instance (PLC.GShow uni, PLC.Closed uni, uni `PLC.Everywhere` PLC.PrettyConst, P
         CompilationError e -> "Unexpected error during compilation, please report this to the Plutus team:" PP.<+> PP.pretty e
         UnsupportedError e -> "Unsupported feature:" PP.<+> PP.pretty e
         FreeVariableError e -> "Reference to a name which is not a local, a builtin, or an external INLINABLE function:" PP.<+> PP.pretty e
+        -- FIXME: transition into an sdoc solution for prettyprinting, to be compatible with ghc and its plugins
```

Comment:
I don't think we should do this. GHC's `SDoc` is a huge pain, and the sooner we can get rid of it the better.